README.md

```diff
@@ -34,14 +34,24 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. If you were to describe semantic HTML to the next cohort of students, what would you say?
 
+semantic explains what it does for people reading your code / organizes the project. not meant for aesthetics.
+
```

Comment:
Very true, SEO and accessibility will also be major everyday reasons why you use semantic code -> https://developer.mozilla.org/en-US/docs/Learn/Accessibility/HTML